stubs/homeassistant/const.pyi

```diff
@@ -1,6 +1,7 @@
 ATTR_ENTITY_ID: str
 ATTR_HIDDEN: str
 ATTR_TEMPERATURE: str
+ATTR_PRESSURE: str
```

Comment:
There is no `ATTR_PRESSURE` constant in https://github.com/home-assistant/core/blob/dev/homeassistant/const.py (the stubs match up to the corresponding files in homeassistant/core).

Reply:
Thanks :D ill try to get this done today